app/src/main/res/values-ar/strings.xml

```diff
@@ -90,6 +90,7 @@
     <string name="equalizer">Equalizer</string>
     <string name="swap_prev_next">Swap previous/next headphone buttons</string>
     <string name="exclude_folder_description">Excluding a folder will prevent the displaying of all audio files within. Files manually added to playlists will not be affected.</string>
+    <string name="gapless_playback">Gapless playback</string>
```

Comment:
it was enough to add the new string into the default values/strings.xml, while the PR gets accepted

Reply:
I'll keep that in mind :)